.changeset/eight-insects-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': patch
+---
+
+Add `useContent` hook to have a reference to the current main content element
```

Comment:
I believe we need a changeset for search too?

Reply:
updated, thanks